src/libraries/System.Private.CoreLib/src/System/Activator.RuntimeType.cs

```diff
@@ -118,32 +116,18 @@ public static partial class Activator
                                                            object?[]? activationAttributes,
                                                            ref StackCrawlMark stackMark)
         {
-            Type? type = null;
-            Assembly? assembly = null;
+            Assembly assembly;
             if (assemblyString == null)
             {
                 assembly = Assembly.GetExecutingAssembly(ref stackMark);
             }
             else
             {
                 AssemblyName assemblyName = new AssemblyName(assemblyString);
-
-                if (assemblyName.ContentType == AssemblyContentType.WindowsRuntime)
-                {
-                    // WinRT type - we have to use Type.GetType
-                    type = Type.GetType(typeName + ", " + assemblyString, throwOnError: true, ignoreCase);
-                }
-                else
-                {
-                    // Classic managed type
-                    assembly = RuntimeAssembly.InternalLoad(assemblyName, ref stackMark, AssemblyLoadContext.CurrentContextualReflectionContext);
-                }
+                assembly = RuntimeAssembly.InternalLoad(assemblyName, ref stackMark, AssemblyLoadContext.CurrentContextualReflectionContext);
             }
 
-            if (type == null)
-            {
-                type = assembly!.GetType(typeName, throwOnError: true, ignoreCase);
-            }
+            Type? type = assembly.GetType(typeName, throwOnError: true, ignoreCase);
```

Comment:
If this code wasn't needed anymore, is https://github.com/dotnet/runtime/blob/4c92aef2b08f9c4374c520e7e664a44f1ad8ce56/src/libraries/System.Private.CoreLib/src/System/Reflection/AssemblyNameFormatter.cs#L74-L75 still needed?

Reply:
I believe that can also be removed.

Reply:
I think that one is reachable:

```csharp
using System;

Console.WriteLine(AssemblyName.GetAssemblyName(@"C:\Windows\System32\WinMetadata\Windows.UI.winmd"));
```

```
Windows.UI, Version=255.255.255.255, Culture=neutral, PublicKeyToken=null, ContentType=WindowsRuntime
```